server/parsec/components/user.py

```diff
@@ -46,9 +46,18 @@ class UserDump:
     human_handle: HumanHandle
     created_on: DateTime
     revoked_on: DateTime | None
-    current_profile: UserProfile
+    # Ordered oldest to newest
+    profile_updates: list[tuple[DateTime, UserProfile]]
     devices: list[DeviceID]
 
+    @property
+    def current_profile(self) -> UserProfile:
+        return self.profile_updates[-1][1]
+
+    @property
+    def initial_profile(self) -> UserProfile:
+        return self.profile_updates[0][1]
+
```

Comment:
Not related to sequester, extract it to another PR? (see in `memory/user.py`)

Reply:
Moved in #9451